### PR TITLE
Break out of correct loop

### PR DESCRIPTION
**Pull request by neekolas**
## Summary

I realized that once the notifications listener has an issue with a stream, it will fail indefinitely. Looks like this is related to not resetting the connection.

Also fixes a bug in the query for subscriptions

### CODE REVIEW

File: pkg/xmtp/listener.go [hunk 1]
```diff
@@ -84,12 +85,12 @@ func (l *Listener) startMessageListener() {
 				msg, err := stream.Recv()
 				if err == io.EOF {
 					l.logger.Info("stream closed")
-					break
+					break streamLoop
```
Comment (mkobetic) [inline pkg/xmtp/listener.go:88]: This is weird, which loop was it breaking out of? Couldn't be the outer one, was it?

File: pkg/subscriptions/subscriptions.go [hunk 2]
```diff
@@ -82,11 +82,11 @@ func (s SubscriptionsService) Unsubscribe(ctx context.Context, installationId st
 
 func (s SubscriptionsService) GetSubscriptions(ctx context.Context, topic string) (out []interfaces.Subscription, err error) {
 	results := make([]db.Subscription, 0)
-	_, err = s.db.NewSelect().
+	err = s.db.NewSelect().
 		Model(&results).
 		Where("topic = ?", topic).
-		Where("is_active = ?", true).
-		Exec(ctx)
+		Where("is_active = TRUE").
```
Comment (mkobetic) [inline pkg/subscriptions/subscriptions.go:88]: Why this change? Binding booleans doesn't work?

**Review by mkobetic — APPROVED** [review]

File: pkg/xmtp/listener.go (hunk 1 above)
Comment (neekolas) [reply pkg/xmtp/listener.go:88]: No, it was breaking out of the select

File: pkg/subscriptions/subscriptions.go (hunk 2 above)
Comment (neekolas) [reply pkg/subscriptions/subscriptions.go:88]: No real reason. Binding booleans totally works, but isn't necessary given this is a constant
